tsconfig.json

```diff
@@ -28,7 +28,6 @@
     "outDir": "./build/",
     "preserveConstEnums": true,
     "removeComments": true,
-    // "sourceMap": true,
```

Comment:
? Is this a temporary remove?

Reply:
No, I marked it as a comment when implementing the Cypress test. Cypress couldn't work with it and requested to remove it. Dominic told me to remove it completely in the next ticket.

Reply:
Ok, we should discuss to move our e2e tests to a separate project, so it does not conflict with our development setup. This would also be a preparation to move the actual tests from running on the jenkins to a separate test server.